tests/unit/helpers/MigrationHelperTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace prime\tests\helpers;
+
+use prime\helpers\MigrationHelper;
+use yii\base\NotSupportedException;
+use yii\db\Migration;
+
+final class MigrationHelperTest extends \Codeception\Test\Unit
+{
+    private Migration $migration;
```

Comment:
don't use the migration for changing database stuff.
Use `\Yii::$app->db` to interact with the database. It has similar commands.